installscripts/wizard/scenarios/jazz_generic_details.py

```diff
@@ -66,18 +75,21 @@ def get_stack_generic_details(jazz_branch):
 
     region = raw_input("AWS Region (e.g. us-east-1): ")
     if region not in knownWorkingRegions:
-        print('Warning: This installer has not been tested against the region you specified.\nPlease check the Jazz documentation (https://github.com/tmobile/jazz-installer/wiki#prerequisites) to verify the region you have chosen supports the required AWS resources.')
+        print(
+            'Warning: This installer has not been tested against the region you specified.\nPlease check the Jazz documentation (https://github.com/tmobile/jazz-installer/wiki#prerequisites) to verify the region you have chosen supports the required AWS resources.'
```

Comment:
Can we also put a user prompt and have them confirm if they want to continue?

Reply:
Yep.